### DISCUSSION TITLE
BubbleMessageComponent()

### DISCUSSION THREAD

**Discussion by dnaldoog** [RomanKubiak/ctrlr#344] [Show and tell]
I found this in one of Roman's panels **Demo - Request Program**.

I didn't know that bubble message popups are possible.

----

[BubbleMessageComponent_1.0.0.2021-08-04_09-04.panel.zip](https://github.com/RomanKubiak/ctrlr/files/6927687/BubbleMessageComponent_1.0.0.2021-08-04_09-04.panel.zip)

![Demo-Request Program](https://user-images.githubusercontent.com/76831683/128101638-4093968c-78fd-4bd6-bb14-3e4123c8a00c.png)

**Comment by damiensellier** [comment]
That's interesting!

**Reply by bijlevel** [reply]
I did once wonder about bubbles too and remember asking about it on the old forum. I believe Roman explained it to me, so I knew  wat bubbles mean, but forgot about it. But it can be very useful and thinking about it I probable will use it for our Kurzweil panels... Thanks for mentioning it!

**Reply by dnaldoog** [reply]
I seem to remember reading it wasn’t possible or broken too but it works fine. I have started using it in a panel. I  am using it to give hints on some uiSliders e.g “double click for default value” and then I set it to only show once. Very useful! :)

**Reply by damiensellier** [reply]
There was such bubbles in older versions of Arturia V collection to explain the different functions of the synths, it worked as  a step by step mini tutorial.

**Reply by bijlevel** [reply]
I did load the panel, but the bubbles appear ONLY (and then all at once) when I enter the big blue rectangle. Nothing happens when I enter a button. Or is this the way it should behave in this particular example? (This is with version 5.3.198 mac and v5.6.23 mac.)

**Reply by dnaldoog** [reply]
Oh that’s the way it supposed to be - by moving the mouse over the big blue rectangle it triggers the bubble methods on different objects .

**Reply by bijlevel** [reply]
Ah, thanks!
